src/CBT.Microsoft.CSharp.targets/Module/Build.props

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
+  <PropertyGroup Condition=" '$(MSBuildProjectExtension)' == '.csproj' Or '$(ForceImportCBTMicrosoftCSharpTargets)' == 'true' ">
```

Comment:
For property naming convention we may want to consider a pattern.   Such as if it turns on a feature/module for a project it would start with enable.  If it uses a certain switch or whatever for that given module it would start with use.  If it causes something to run it would start with run (in this case usually that thing must be enabled before it can run).

Anyway I bring this up because if we use a naming convention for properties we intend to expose for users then the cbt vsix or even the user can easily identify said properties and allow them to make a judgement call on if the intent is for them to be able to tweak these properties or if they are just hacking how something uses.